tools/base.tsconfig.json

```diff
@@ -16,11 +17,14 @@
     "noUncheckedIndexedAccess": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "strictNullChecks": true,
     "esModuleInterop": true,
     "moduleResolution": "node",
     "erasableSyntaxOnly": true,
     "skipLibCheck": true,
+    "composite": true,
```

Comment:
If we do want to use composite (even if we can't take advantage of `.tsbuildinfo`), we can fold its uses in the tsconfig files that inherit from this.